instruqt/custom-fields/03-updating-the-application/assignment.md

```diff
@@ -18,60 +21,103 @@ tabs:
   type: code
   hostname: shell
   path: /home/replicant/
+- title: Vendor
+  type: website
+  url: https://vendor.replicated.com
+  new_window: true  
 difficulty: basic
 timelimit: 600
 ---
 
-## Setting Up Dev Environment ##
+## Setting Up the Dev Environment ##
 
-Before we update the application we need to make sure we have a dev environment set up. We have provided a dev environment for you to use but you could also run the `replicated` command line from other Linux or Mac machine like your laptop. To use the dev environment we provided for you click on the **Dev** tab.
+Before we update the application, let's set up our dev environment. In this lab, you will notice some new tabs:
 
-Make sure to set the `REPLICATED_APP` and `REPLICATED_API_TOKEN` environment variables. This is covered in the Replicated CLI track in more detail.
+* **Dev:** This is a Linux environment preloaded with some handy tools, like the `replicated` command line. This will be our dev enviornment.
```

Comment:
```suggestion
* **Dev:** This is a Linux environment preloaded with some handy tools, like the `replicated` command line. This will be our dev environment.
```